### PR TITLE
Improve `InsufficientTxInputs` error message

### PR DESCRIPTION
**Pull request by rmgaray**
This PR improves error messages whenever an `InsufficientTxInputs` exception is thrown due to not being enough ADA in the user's wallet for covering either of these:

1. TX fees and min. ADA for all the unbalanced TX's outputs
2. UTxOs created during balancing

### CODE REVIEW

File: frontend/src/Utils.purs [hunk 1]
```diff
@@ -586,7 +586,58 @@ repeatUntilConfirmed timeout maxTrials contract = do
...
+  result <- balanceTx ubTx
+  bTx <- case result of
+    Left e -> handleError e
+    Right r -> pure r
+  signTransaction bTx
+  where
+  handleError :: forall a. BalanceTxError -> Contract r a
+  handleError = case _ of
+    InsufficientTxInputs expected actual
+      | adaAmt expected < adaAmt actual ->
```
Comment (nazrhom) [inline frontend/src/Utils.purs:606]: Maybe it is not worth splitting between the 2 cases in `InsufficientTxInputs`. I would rather have a nice way to render what is `expected` vs what is `actual` showing both ada and non-ada asset differences. Maybe even some thing like you need `expected - actual` more in inputs

Comment (rmgaray) [reply frontend/src/Utils.purs:606]: I think it makes sense. I guess the original idea was to distinguish between "not enough money to _spend_" and "not enough money to _operate_". But if final users will be reading this, this distinction is probably alien to them. Just showing what needs to be done to fix it is probably better.

**Comment by nazrhom** [comment]
hey @rmgaray I see more changes in the PR that the description, is it all intended?

**Comment by rmgaray** [comment]
Hey @nazrhom, I just fixed that. I made a mistake while rebasing apparently and for some reason it filled the history with commits from previously merged PRs. Now you should see only three commits.

**Review by nazrhom — APPROVED** [review]

**Pull request merged**